content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,107 @@
+- Training cards dialog
+  - [NEW] `training_cards.confirm.title`: used for the modal title
+- Confirmation when verifying a report dialog
+  - [NEW] `confirm.verification.title`: used for the modal title
+  - [NEW] `confirm.verification.submit`: used for the modal submit button label.
+
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+
+
+## Highlights
+
+### Store purged docs cache in couchdb
+
+Each user gets their own doc in a new database `medic-cache` to store purged document ids the database is destroyed (deleted and recreated) every time purging runs
+
+### Can't turn off permissions for `national_admin` role in admin app.
+
+Removes hardcoded conditions making national_admin role sometimes behave as CouchDb _admin.
+Adds `national_admin` as role in default config for backwards compatibility.
+
+### Features
+
+None.
+
+### Improvements
+
+- [#6514](https://github.com/medic/cht-core/issues/6514): Migrate from SW-Precache to Workbox
+- [#8179](https://github.com/medic/cht-core/issues/8179): nginx and haproxy should respect `Accept` headers when responding with error templates
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+- [#8442](https://github.com/medic/cht-core/issues/8442): Telemetry geolocation:failure events for geolocation should be unique
+- [#8498](https://github.com/medic/cht-core/issues/8498): Remove targets disable admin option
+- [#8506](https://github.com/medic/cht-core/issues/8506): Update browser support notice to match official versions
+- [#8511](https://github.com/medic/cht-core/issues/8511): Update configuration deployment action to use latest cht-conf
+
+### Security fixes
+
+- [#8443](https://github.com/medic/cht-core/issues/8443): Masking credentials
+
+### Performance improvements
+
+- [#8238](https://github.com/medic/cht-core/issues/8238): Figure out a different way to use purged docs cache
+- [#8451](https://github.com/medic/cht-core/issues/8451): Add support for HTTP/2 in nginx 
+
+### Bug fixes
+
+- [#6180](https://github.com/medic/cht-core/issues/6180): Any text containing back ticks looks weird in the webapp
+- [#8081](https://github.com/medic/cht-core/issues/8081): Add contact_type and place_id to contacts exported by `/api/v2/export/contacts`
+- [#8243](https://github.com/medic/cht-core/issues/8243): Can't turn off permissions for `national_admin` role in admin app
+- [#8490](https://github.com/medic/cht-core/issues/8490): API crashes when CouchDB is overloaded
+- [#8527](https://github.com/medic/cht-core/issues/8527): Unable to build nginx with symlinks in some environments
+
+### Technical improvements
+
+- [#6132](https://github.com/medic/cht-core/issues/6132): Files containing tests should use the .spec.js extension
+- [#6289](https://github.com/medic/cht-core/issues/6289): Upgrade to CouchDB v3
```

Comment:
This should be a highlight and a performance improvement (not a technical improvement)... I've updated the issue labels to reflect this.

Reply:
Ticket `#6289` was moved to the `Performance improvements` section.
I asked @dianabarsan's help to add a more details explanation of the ticket to add it in the `Highlights` section